src/components/localization/localized-link.js

```diff
@@ -101,7 +101,7 @@ export const LocalizedLink = React.forwardRef(({ to, ...props }, ref) => {
     // Because otherwise this would add a trailing slash
     const { is_default, path, affiliate_lang } = language_config[locale]
     const is_non_localized = non_localized_links.includes(to)
-    const path_to = is_default || is_non_localized ? to : `/${path}${is_index ? `` : `${to}`}/`
+    const path_to = is_default || is_non_localized ? to : `/${path}${is_index ? `` : `${to}`}`
```

Comment:
please help to make utility function to normalize path that is passed to localized 🙏

Reply:
Hi @mustofa-binary , I have made the changes. Let me know, if it exactly what you meant.